linkerd/metadata.csv

```diff
@@ -38,3 +39,31 @@ linkerd.rt.client.request_latency_ms.quantile,gauge,,millisecond,,Stats of the l
 linkerd.rt.server.request_latency_ms.quantile,gauge,,millisecond,,Stats of the latency of requests in milliseconds for the server,-1,linkerd,server requests latency
+linkerd.request_total,count,,,,Total count of HTTP requests.,0,linkerd,
+linkerd.response_latency,gauge,,millisecond,,Elapsed times between a request's headers being received and its response stream completing,0,linkerd,
+linkerd.response_total,count,,,,Total count of HTTP responses.,0,linkerd,
+linkerd.retry_skipped_total,count,,,,Total count of retryable HTTP responses that were not retried.,-1,linkerd,
+linkerd.route.request_total,count,,,,Total count of HTTP requests.,0,linkerd,
+linkerd.route.response_latency,gauge,,millisecond,,Elapsed times between a request's headers being received and its response stream completing,0,linkerd,
+linkerd.route.response_total,count,,,,Total count of HTTP responses.,0,linkerd,
+linkerd.route.retry_skipped_total,count,,,,Total count of retryable HTTP responses that were not retried.,-1,linkerd,
+linkerd.route.actual_request_total,count,,,,Total count of HTTP requests.,0,linkerd,
+linkerd.route.actual_response_latency,gauge,,millisecond,,Elapsed times between a request's headers being received and its response stream completing,0,linkerd,
+linkerd.route.actual_response_total,count,,,,Total count of HTTP responses.,0,linkerd,
+linkerd.route.actual_retry_skipped_total,count,,,,Total count of retryable HTTP responses that were not retried.,-1,linkerd,
+linkerd.tcp.open_total,count,,,,Total count of opened connections,0,linkerd,
+linkerd.tcp.open_connections,gauge,,,,Number of currently-open connections,0,linkerd,
+linkerd.tcp.read_bytes_total,gauge,,byte,,Total count of bytes read from peers,0,linkerd,
+linkerd.tcp.write_bytes_total,gauge,,byte,,Total count of bytes written to peers,0,linkerd,
+linkerd.tcp.close_total,count,,,,Total count of closed connections,0,linkerd,
+linkerd.tcp.connection_duration,gauge,,,,Connection lifetimes,0,linkerd,
+linkerd.control.request_total,count,,,,Total count of HTTP requests.,0,linkerd,
+linkerd.control.response_latency,gauge,,millisecond,,Elapsed times between a request's headers being received and its response stream completing,0,linkerd,
+linkerd.control.response_total,count,,,,Total count of HTTP responses.,0,linkerd,
+linkerd.control.retry_skipped_total,count,,,,Total count of retryable HTTP responses that were not retried.,-1,linkerd,
+linkerd.process.start_time,gauge,,,,Time that the process started (in seconds since the UNIX epoch),0,linkerd,
+linkerd.process.cpu_seconds_total,count,,,,Total user and system CPU time spent in seconds.,0,linkerd,
+linkerd.process.open_fds,gauge,,,,Number of open file descriptors.,0,linkerd,
+linkerd.process.max_fds,gauge,,,,Maximum number of open file descriptors.,0,linkerd,
+linkerd.process.virtual_memory,gauge,,byte,,Virtual memory size in bytes.,0,linkerd,
+linkerd.process.resident_memory,gauge,,byte,,Resident memory size in bytes.,0,linkerd,
```

Comment:
Seems the `short_name` is missing for all new metrics (and some of the existing metrics).

Reply:
Yeah sometimes it's not worth the effort if the metric is descriptive enough

Reply:
Yeah in this case it feels somewhat repetitive.